be/src/common/config.h

```diff
@@ -589,6 +589,8 @@ CONF_mInt64(max_runnings_transactions_per_txn_map, "100");
 // The tablet map shard size, the value must be power of two.
 // this is an enhancement for better performance to manage tablet.
 CONF_Int32(tablet_map_shard_size, "32");
+// The value must be power of two.
+CONF_Int32(pk_index_map_shard_size, "32");
```

Comment:
Make this large, it won't cost too much memory. How about 4096?

Reply:
Ok，but why `tablet_map_shard_size` just be set to 32 by the way, and I think the the memory consumption is fixed size about the number fo tablet